docs/wetboek.md

```diff
@@ -145,19 +145,28 @@
 ### Artikel I-1 Diefstal (310 SR)
 
 1. Schuldig is een persoon die een goed, of voertuig, wegneemt zonder daarvoor toestemming of enig recht toe te hebben
-2. Bij feiten hieronder vemeld worden verzwarende omstandigheden in rekening gebracht met de daarbij horende strafmaat. in dit geval zal de straf die geëist wordt vermeerderd worden met 33%:
-3. Wanneer een diefstal gedurende voor de nachtrust bestemde tijd, in een woning of op een besloten erf waarop een woning staat, door iemand die aldaar buiten weten of tegen de wil van de rechthebbende bevindt. en/of;
-4. Wanneer diefstal die door twee of meerdere verenigde personen. en/of;
-5. Wanneer diefstal waarbij de schuldige zich de toegang tot de plaats van het misdrijf heeft verschaft of het weg te nemen goed onder zijn bereik heeft gebracht door middel van braak, verbreking of inklimming, van valse sleutels, van een valse order of een vals kostuum.
-6. Bij een poging tot diefstal is de maximaal op te leggen straf 2/3 van de volledige strafeis
+2. Bij een poging tot diefstal is de maximaal op te leggen straf 33% van de volledige strafeis
 
 |   | *Celstraf*  | *Taakstraf*  | *boete*  |
 |---|---|---|---|
-|  **Eerste Veroordeling** |   | 20 uur  | €2,000  |
-| **Tweede Veroordeling**  | 18 maanden  |  | €4,000  |
-| **Meerdere Veroordelingen**  | 36 maanden  |   | €6,000  |
+|  **Eerste Veroordeling** |   | 20 uur  | €10000,-  |
+| **Tweede Veroordeling**  | 18 maanden  |  | €12500,-  |
+| **Meerdere Veroordelingen**  | 36 maanden  |   | €15000,-  |
 
-### Artikel I-2 Diefstal met geweld / Overval (312 SR)
+### Artikel I-2 Gekwalificeerde Diefstal (311 SR)
+
+1. Schuldig is een persoon die een diefstal pleegt gedurende de voor de nachtrust bestemde tijd, in een woning of op een besloten erf waarop een woning staat, door iemand die zich aldaar buiten weten of tegen de wil van de rechthebbende bevindt. 
+2. Schuldig is een persoon die een diefstal pleegt in vereniging. 
+3. Schuldig is een persoon die een diefstal pleegt waarbij de schuldige zich de toegang tot de plaats van het misdrijf heeft verschaft of het weg te nemen goed onder zijn bereik heeft gebracht door middel van braak, verbreking of inklimming, van valse sleutels, van een valse order of een vals kostuum;
+4. Bij een poging tot gekwalificeerde diefstal wordt de straf met 33% verminderd.
+
+|   | *Celstraf*  | *Taakstraf*  | *boete*  |
+|---|---|---|---|
+|  **Eerste Veroordeling** |   | 50 uur  | €15000,-  |
```

Comment:
kosten pagina ook updaten

Reply:
Ik vind persoonlijk dat de verhouding in straf tussen deze twee artikelen en Diefstal met geweld / Overval wel erg klein wordt. De geldboetes zijn bijna gelijk aan gewapende overal...

Reply:
Dit is in overleg gedaan. In nederland staat er een maximum op van 5e categorie, wat op 90k uit zo komen.